tests/commons/src/main/java/org/codice/ddf/admin/comp/graphql/GraphQlHelper.java

```diff
@@ -57,6 +59,14 @@ public GraphQLRequest createRequest() {
     return new GraphQLRequest(queryResourcePath, mutationResourcePath, graphQlEndpoint);
   }
 
+  public GraphQLRequest createMutationRequest(String usingMutation, Map<String, Object> arguments) {
+    return populateRequestArgs(createRequest().usingMutation(usingMutation), arguments);
+  }
+
+  public GraphQLRequest createQueryRequest(String usingQuery, Map<String, Object> arguments) {
+    return populateRequestArgs(createRequest().usingQuery(usingQuery), arguments);
```

Comment:
With the other comments I mentioned, I wonder if these methods will even be worth keeping around if we use a builder pattern in `mutationRequest` and `queryRequest`